scene/gui/label.cpp

```diff
@@ -453,6 +455,7 @@ void Label::regenerate_word_cache() {
 			line_width += char_width;
 			total_char_cache++;
 		}
+		page_chars = total_char_cache;
```

Comment:
Why are page_chars set both here, in `regenerate_word_cache`, and in `NOTIFICATION_DRAW`?

Reply:
The one in the `regenerate_word_cache` is just for the initialization, this value would be incorrect. But it will be updated in `NOTIFICATION_DRAW` later. This happens because I can't get the number of chars per line in the prior function.